tests/CMakeLists.txt

```
@@ -44,7 +45,7 @@ set(ENSMALLEN_TESTS_SOURCES
     svrg_test.cpp
     swats_test.cpp
     wn_grad_test.cpp
-)
+    )
```

No need to indent here. :)
```suggestion
)
```